spec/ddtrace/contrib/active_record/tracer_spec.rb

```diff
@@ -41,9 +37,6 @@
     it_behaves_like 'measured span for integration', false
 
     it 'calls the instrumentation when is used standalone' do
-      # expect service and trace is sent
-      expect(spans.size).to eq(1)
-
```

Comment:
A lot of this logic can now be removed because we are not using the global helpers `#span` and `#spans`.

`#span`, for example, asserts that there's only one span available, as to not deceive the developer by returning the first among many when that might not be the intention.